samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/select/SelectActionListener.java

```diff
@@ -785,6 +812,24 @@ private Date getFeedbackDate(Long publishedAssessmentId, Map publishedAssessment
       return null;
   }
 
+  private Date getFeedbackEndDate(Long publishedAssessmentId, Map publishedAssessmentHash){
+    PublishedAssessmentFacade p = (PublishedAssessmentFacade)publishedAssessmentHash.
+        get(publishedAssessmentId);
+    if (p!=null)
+      return p.getFeedbackEndDate();
+    else
+      return null;
+  }
+
+  private Double getFeedbackScoreThreshold(Long publishedAssessmentId, Map publishedAssessmentHash){
+    PublishedAssessmentFacade p = (PublishedAssessmentFacade)publishedAssessmentHash.
+        get(publishedAssessmentId);
+    if (p!=null)
+      return p.getFeedbackScoreThreshold();
```

Comment:
Need some braces to make the blocks clearer

Reply:
yeah, I don't know why the hell I'm imitating the same indentation and bad patterns....looks like I've spent much time working with Samigo......

I changed the implementation to this:

return p != null ? p.getFeedbackEndDate() : null;